parkings/tests/api/operator/test_permit.py

```diff
@@ -0,0 +1,270 @@
+from django.urls import reverse
+from rest_framework.status import (
+    HTTP_200_OK, HTTP_201_CREATED, HTTP_204_NO_CONTENT, HTTP_404_NOT_FOUND)
+
+from parkings.models import Permit, PermitArea
+
+from ....factories.permit import (
+    generate_areas, generate_external_ids, generate_subjects)
+from ..enforcement.test_check_parking import create_area_geom
+
+list_url = reverse('operator:v1:permit-list')
+
+
+def _get_detail_url(obj):
+    return reverse('operator:v1:permit-detail', kwargs={'pk': obj.pk})
+
+
+def _get_permit_data(permit_series):
+    return {
+        'series': permit_series.id,
+        'external_id': generate_external_ids(),
+        'subjects': generate_subjects(),
+        'areas': generate_areas(),
+    }
+
+
+def _check_response(data, obj):
+    permit_response_keys = {
+        'series',
+        'domain',
+        'external_id',
+        'id',
+        'subjects',
+        'areas',
+    }
+
+    assert set(data.keys()) == permit_response_keys
+    assert obj.series.pk == data['series']
+    assert obj.domain.code == data['domain']
+    assert obj.external_id == data['external_id']
+    assert obj.id == data['id']
+    assert obj.subjects == data['subjects']
+    assert obj.areas == data['areas']
+
+
+def test_operator_can_create_permit_with_valid_post_data(
+    operator_api_client, permit_series, enforcement_domain
+):
+    permit_data = _get_permit_data(permit_series)
+    permit_data.update(domain=enforcement_domain.id)
```

Comment:
The endpoint should take the **code** of the Enforcement Domain, not its id in the `domain` field. And since all operator permit endpoints will need the domain in the data, you could as well hard code it in `_get_permit_data` as `'domain': 'ED1',` or something similar (and make sure that the enforcement domain code is ED1).